packages/zoe/package.json

```diff
@@ -63,6 +63,7 @@
     "ses": "^0.11.0"
   },
   "files": [
+    "bundles/",
```

Comment:
I didn't think that we wanted to package the bundles.  Is there a reason we want to?

Reply:
Build steps are okay when you've got a fresh Git repo and are developing the packages in question, but for NPM tarball packages, the build products are typically installed as part of the package unless they are platform-dependent (in which case they're built at `install` time).

The bundles need to be built before Zoe can be used (and they are platform-independent), so by my reasoning above, they should be put in the NPM package.

Reply:
Sounds good to me!